apps/src/clientApi.js

```diff
@@ -66,13 +66,15 @@ ApiRequestHelper.prototype.get = function (localUrl, callback) {
   $.ajax({
     url: this.apiBaseUrl_ + localUrl,
     type: 'get',
-    dataType: 'json'
-  }).done(function (data /*, textStatus, jqXHR*/) {
-    callback(null, data);
-  }).fail(function (jqXHR, textStatus, errorThrown) {
-    callback(
-        new Error('textStatus: ' + textStatus + '; errorThrown: ' + errorThrown),
-        null);
+    dataType: 'json',
+    success: function (data /*, textStatus, jqXHR*/) {
```

Comment:
Realize this isn't new, but I personally dislike having commented out/unused args. Would rather just see success: function (data) {